### PR TITLE
EJ2-Documentation-887110-Ug-Documentation-For-Edit-types

### PR DESCRIPTION
**Pull request by harinikrh**

### CODE REVIEW

File: ej2-javascript/grid/editing/edit-types.md [hunk 1]
```diff
@@ -101,26 +152,52 @@ In the below demo, while editing the row we have restricted to type the decimal
...
-### Provide custom data source and enabling filtering to DropDownList
+## Customize DropDownList control of DropDownEdit type 
 
-You can provide data source to the DropDownList by using the [`columns.edit.params`](../../api/grid/column/#edit) property.
+You can customize the `DropDownList` control in Grid edit form using its property. This customization allows you to configure various properties of the DropDownList, tailoring its behavior and appearance to match your specific requirements within the Grid. The behavior of the editor control can be fine-tuned through the [columns->edit->params](../../api/grid/column/#edit) property. 
 
-While setting new data source using edit params, you must specify a new `query` property too for the dropdownlist as follows,
+Control|Edit Type |Description| Example Customized edit params
+-----|-----|-----|----|
+[DropDownList](../../api/drop-down-list/)-|DropDownEdit|  The `dropdownedit` type renders a DropDownList control for string data type columns. To customize the DropDownList control, refer to the [DropDownList API documentation](../../api/drop-down-list) for detailed information on available properties.  | params: { value: ‘Germany’ }
```
Comment (VigneshNatarajan27) [inline ej2-javascript/grid/editing/edit-types.md:161]: remove unnecessary hypen used here.

File: ej2-javascript/grid/editing/edit-types.md [hunk 2]
```diff
@@ -225,9 +298,245 @@ Use the cell edit template feature of the Grid to render the TimePicker componen
...
+The following example demonstrates how to prevent the enter key functionality in multiline textbox during editing by utilizing the `focus` event:
+
+{% if page.publishingplatform == "typescript" %}
+
+ {% tabs %}
+{% highlight ts tabtitle="index.ts" %}
+{% include code-snippet/grid/edit-textbox-control/index.ts %}
+{% endhighlight %}
+{% highlight html tabtitle="index.html" %}
+{% include code-snippet/grid/edit-textbox-control/index.html %}
```
Comment (VigneshNatarajan27) [inline ej2-javascript/grid/editing/edit-types.md:513]: modify in same js and ts samples

DisableHtmlEncode is not defined

File: ej2-javascript/grid/editing/edit-types.md [hunk 3]
```diff
@@ -225,9 +298,245 @@ Use the cell edit template feature of the Grid to render the TimePicker componen
...
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "page.domainurl/code-snippet/grid/edit-textbox-control" %}
+{% endif %}
+
+### Render AutoComplete control in edit form
```
Comment (VigneshNatarajan27) [inline ej2-javascript/grid/editing/edit-types.md:533]: remove any one AutoComplete topic

File: ej2-javascript/grid/editing/edit-types.md [hunk 4]
```diff
@@ -256,71 +565,264 @@ Use the cell edit template feature of the Grid to render the AutoComplete compon
 {% previewsample "page.domainurl/code-snippet/grid/grid-cs30" %}
 {% endif %}
 
-### Render MultiSelect DropDown component while editing
+### Render MaskedTextBox control in edit form 
+
+The Syncfusion Grid allows you to render a MaskedTextBox control within the Grid's edit form for a specific column. This feature is especially useful when you need to provide masked input fields that require a specific format, such as phone numbers or postal codes.
 
-Use the cell edit template feature of the Grid to render the MultiSelect DropDown component in the Grid edit form. In the below sample, we have rendered MultiSelect DropDown component in the **ShipCity** column.
+To render a MaskedTextBox control in the edit form, you need to define a cell edit template for the column using [edit](../../api/grid/column/#edit) property. The cell edit template that used as an editor for a particular column.
```
Comment (VigneshNatarajan27) [inline ej2-javascript/grid/editing/edit-types.md:572]: the edit property specifies the cell edit template that used

File: ej2-javascript/grid/editing/edit-types.md (hunk 1 above)
Comment (harinikrh) [reply ej2-javascript/grid/editing/edit-types.md:161]: removed the hypen

File: ej2-javascript/grid/editing/edit-types.md (hunk 2 above)
Comment (harinikrh) [reply ej2-javascript/grid/editing/edit-types.md:513]: added the property

File: ej2-javascript/grid/editing/edit-types.md (hunk 3 above)
Comment (harinikrh) [reply ej2-javascript/grid/editing/edit-types.md:533]: removed the topic

**Review by VigneshNatarajan27 — APPROVED** [review]
changes are fine

**Pull request merged**